FirstApp/CellTableViewCell.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  CellTableViewCell.swift
+//  FirstApp
+//
+//  Created by Akua Afrane-Okese on 2020/02/19.
+//  Copyright © 2020 Akua Afrane-Okese. All rights reserved.
+//
+
+import UIKit
+
+class CellTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var recipeImageView: UIImageView!
+    @IBOutlet weak var titleLabel: UILabel!
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        // Initialization code
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+
+        // Configure the view for the selected state
+    }
```

Comment:
I would say that these functions are redundant as they simply override and then call the super with the exact same arguments.

Reply:
The functions were made on creation of the file. I will check if everything is fine if removed. If so, then I will permanently remove them. Otherwise they will stay.🧐